test/test_model.py

```diff
@@ -5,7 +5,12 @@
 import juju.model
 
 
+# TODO: It would be better if this implemented these as executables that were called
+# via the actual juju.model.ModelBackend implementation.
```

Comment:
... or at least had some simple smoke tests for that. We can still keep the current test backend if it turns out it's easier to test more deeply using it. We just need some sort of verification that our real backend has something the model can indeed use.